content/docs/dataform-web/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Dataform Web
+---
+
+Dataform Web provides a fully managed environment for Dataform projects, where your data team can work together to develop your core analytics infrastructure.
+
+![hero](https://static.dataform.co/images/hero_screenshot.png)
+
+It provides a number of features on top of the open-source framework that makes it a great choice for larger teams.
+
+To get started with Dataform Web, check out our [getting started tutorial](tutorials/101) or [sign up](https://dataform.co) and jump straight into a trial.
```

Comment:
This link is broken, think it should be `dataform-web/tutorials/101` works but a relative dir would be better